data/yaml/venues/neusymbridge.yaml

```diff
@@ -0,0 +1,4 @@
+acronym: NeusymBridge
+name: 'Proceedings of NeusymBridge 2024: The First Workshop on Bridging Neurons and
```

Comment:
The venue name is not "Proceedings of" — that's the book title. Shouldn't have "first" or 2024 or LREC etc

Reply:
Done